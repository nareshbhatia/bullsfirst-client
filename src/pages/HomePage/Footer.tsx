import React from 'react';
import './Footer.css';

export const Footer = () => {
  return (
    <div data-testid="home-footer" className="container footer">
      <div className="max-w-mobile">
        <i>
          <small>
            This is a demo application. All data displayed is fictitious.
          </small>
        </i>
      </div>
    </div>
  );
};
