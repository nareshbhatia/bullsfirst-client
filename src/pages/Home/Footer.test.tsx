import React from 'react';
import { render } from '../../test/test-utils';
import { Footer } from './Footer';

describe('<Footer />', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
