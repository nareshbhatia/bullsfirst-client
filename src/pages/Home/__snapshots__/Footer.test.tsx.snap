// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="container footer"
    data-testid="home-footer"
  >
    <div
      class="max-w-mobile"
    >
      <i>
        <small>
          This is a demo application. All data displayed is fictitious.
        </small>
      </i>
    </div>
  </div>
</DocumentFragment>
`;
